cyclus/cpp_cyclus.pxd

```diff
@@ -655,7 +655,7 @@ cdef extern from "agent.h" namespace "cyclus":
         void EnterNotify()
         void BuildNotify(Agent*)
         void DecomNotify(Agent*)
-        void Decommission()
+        void Decommission() except +
```

Comment:
my cython isn't good enough to know what the correct capitalization is after all the back-and-forth?

Reply:
My understanding is that the pxd file is what makes the c++ functions available to python. So in the pyx file it looks like this
decommission()
         (<somestuff>).Decommission()

The decommission() function is called in python and Decommission() is called through that via cython.